cloud-function/src/app.ts

```diff
@@ -74,7 +74,7 @@ router.get(
   proxyContent
 );
 router.get(
-  "/[^/]+/blog($|/*)",
+  ["/[^/]+/blog($|/*)", "/[^/]+/curriculum($|/*)"],
```

Comment:
Nit: Does this not work?

```suggestion
  "/[^/]+/(blog|curriculum)($|/*)",
```